express/scripts/scripts.js

```diff
@@ -808,6 +808,8 @@ export function decorateBlock(block) {
   const blockName = block.classList[0];
   if (blockName) {
     let shortBlockName = blockName;
+    const section = block.closest('.section');
+    if (section) section.classList.add(`${[...block.classList].join('-')}-container`.replace(/--/g, '-'));
```

Comment:
I think the double-dash replacement is obsolete now.
```suggestion
    if (section) section.classList.add(`${[...block.classList].join('-')}-container`;
```